### PR TITLE
[FIX][IMPROVEMENT][I18N] Adds support for RTL ViewPager for RTL languages

### LINKED ISSUE #2301: [BUG] Emoji Picker ViewPager does not support RTL swiping for RTL languages
## Describe the bug

For RTL languages, emoji categories in the TabLayout change direction, and that is the desired behavior; however, in the Emoji Picker, it does not change the swiping direction with respect to the applied locale.

## To Reproduce

1. Go to any chat.
2. Open Emoji Picker either from the text composer or when reacting to a message.
3. Swipe between the fragments.
4. Swiping direction is not changed.

## Expected behavior

Emoji Picker should support a RTL ViewPager as well.

## Screenshots

Emoji Picker

## Versions

Rocket.Chat.Android version: 3.4.0 (2065)
Rocket.Chat Server version: 1.1.0-develop

### PR DESCRIPTION
**Pull request by hfeky**
@RocketChat/android

Closes #2301

Hopefully this pull request is the last one to fully support RTL languages in Rocket.Chat Android app.

The RTL ViewPager is based on [Duolingo's RtlViewPager](https://github.com/duolingo/rtl-viewpager); however, I rewrote the code in Kotlin, and modified some parts of it.

#### Changes:

1. Added support for RTL ViewPager. Now one can swipe from right to left in Emoji Picker fragments for RTL languages.
2. Made message send icon (`ic_send_24dp.xml`), drawing send icon (`ic_send_black_24dp.xml`) and backspace icon (`ic_backspace_gray_24dp.xml`) be auto-mirrored for RTL languages.
3. Merged _Preferences messages_ with _Settings messages_ in all strings resource files.
4. Corrected some typos and translation mistakes in Arabic strings resource file.

**Note:** There is a known bug, however, as displayed in the GIF, in which sometimes the tabs of Emoji categories are in text instead of icons. I doubt though that this issue has something to deal with the adapter. The adapter is working perfectly. I tried debugging a lot, and tried checking for line 180 in `EmojiKeyboardPopup` and line 67 in `EmojiPickerPopup`, and the resource icon is always set, but does not appear on the layout. I think the issue might be in _lines 177, 178_ in `EmojiKeyboardPopup` and _lines 64, 65_ in `EmojiPickerPopup`; however, that's just a guess. I do not know the issue exactly. I also noticed it depends on the width of the tabs. If you tried rotating your device to landscape mode, things can change here.

#### GIF for the change:

RTL ViewPager

### CODE REVIEW

**Comment by hfeky** [comment]
@filipedelimabrito I wonder what the issue can be. Do you have any idea?

**Comment by philipbrito** [comment]
@HusseinElFeky I'm not sure the issue you are referencing for (couldn't see the tabs of Emoji categories being shown as text instead of icons). Could you please create an issue for this by describing or showing by images about this issue?

This PR is good enough to be blocked by this issue 😁 
Thank you.

File: app/src/main/res/drawable/ic_send_24dp.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="24dp"
     android:height="24dp"
+    android:autoMirrored="true"
```
Comment (philipbrito) [inline app/src/main/res/drawable/ic_send_24dp.xml:4]: 💯 🔝

**Comment by hfeky** [comment]
@filipedelimabrito Yes, they sometimes occur as text and sometimes not, as shown in the GIF. I know that's very weird. I will make sure to debug this issue more. If I couldn't know the reason, I will create an issue for that soon.

**Review by philipbrito — APPROVED** [review]

**Pull request merged**